lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
Thanks, @clrux 
@muhammad-ammar Is this heading going to apply to all error messages relating to creating teams? If yes, then I suggest  "Your team could not be created." 
I think it's fine that this might repeat the first sentence in the error body text.